### PR TITLE
Added Codacy badge & Revive linting

### PR DESCRIPTION
**Pull request by jilleJr**
Related to https://github.com/iver-wharf/iver-wharf.github.io/issues/47

- Added Codacy badge & revive linting
- Fixed var-naming issues
- Updated CHANGELOG.md

I did not touch the "exported" linting error. Leaving that for a different PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,21 +44,47 @@ GET | PROJECT | 125
...
+You can lint all of the above at the same time by running:
+
+```sh
 npm run lint
 
 # Some errors can be fixed automatically. Keep in mind that this updates the
 # files in place.
 npm run lint-fix
-```
+`
```
Comment (fredx30) [inline README.md:84]: Seems like a mistake.
```suggestion
```
```

**Review by fredx30 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (fredx30) [inline README.md]: 3 Ticks here.

Comment (jilleJr) [reply README.md:84]: Like that? :)

````suggestion
```
````

The trick is that a code block can have more than 3 ticks, as long as it ends with the same number of ticks. So to have a code block contain three ticks, then use 4 ticks:

`````md

Like this:

````suggestion
```
````
`````

File: README.md [hunk 2]
```diff
@@ -43,21 +44,47 @@ GET | PROJECT | 125
...
 
 ---
 
 Maintained by [Iver](https://www.iver.com/en).
 Licensed under the [MIT license](./LICENSE).
+```
```
Comment (fredx30) [inline README.md:90]: The code block should end above.
```suggestion
```
```

**Review by fredx30 — APPROVED** [review]

**Review by Pikabanga — APPROVED** [review]

**Pull request merged**